scripts/winesapos-install.sh

```diff
@@ -809,7 +809,39 @@ elif [[ "${WINESAPOS_DE}" == "plasma" ]]; then
         pacman_install_chroot manjaro-kde-settings manjaro-settings-manager-knotifier
         # Install Manjaro specific KDE Plasma theme packages.
         pacman_install_chroot plasma6-themes-breath plasma6-themes-breath-extra breath-wallpapers sddm-breath-theme
-    fi
+
+elif [[${WINESAPOS_BOOTLOADER} == grub]]; then
+   pacman_install_chroot grub os-prober dosfstools xz bash gettext
+   sudo grub-install ${WINESAPOS_DEVICE}
+   GRUB_CFG_PATH="/boot/grub/grub.cfg"
+   touch $GRUB_CFG_PATH
+   echo "set default=0" >> $GRUB_CFG_PATH
+   echo "set timeout=5" >> $GRUB_CFG_PATH
+   echo " " >> $GRUB_CFG_PATH
+   echo "menuentry "winesapOS" {" >> $GRUB_CFG_PATH
+   echo "    set root='hd0,msdos1'" >> $GRUB_CFG_PATH
+   echo "linux /boot/vmlinuz-linux-lts root=/dev/sda1 rw quiet" >> $GRUB_CFG_PATH
+   echo "initrd /boot/initramfs-linux.img" >> $GRUB_CFG_PATH
+   echo "}" >> $GRUB_CFG_PATH
+   echo " " >> $GRUB_CFG_PATH
+   echo "menuentry "winesapOS (Fallback)" {" >> $GRUB_CFG_PATH
```

Comment:
There's a syntax error here:

```
$ bash -n scripts/winesapos-install.sh 
scripts/winesapos-install.sh: line 827: syntax error near unexpected token `('
scripts/winesapos-install.sh: line 827: `   echo "menuentry "winesapOS (Fallback)" {" >> $GRUB_CFG_PATH'
```

I recommend setting up these pre-commit hooks to help you catch syntax errors:

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-beta.2/CONTRIBUTING.md#getting-started

GRUB is already configured throughout the `winesapos-install.sh` script. We definitely don't want to manually configure the `/boot/grub/grub.cfg` file.

For this PR,  you can just have this block do a `echo STUB` or something similar. I'll go through and refactor the GRUB stuff later since I'm more familiar with the code.